models/Review.js

```diff
@@ -0,0 +1,51 @@
+/** @module models/Review
+* The Review Model
+* Schema:
+* author						String			Author of the review. Required.                 //maybe create User model?
+* text							String			Text of the review.
+* score							Integer			Score of the review, goes from 1-5. Required.
+* target						ObjectID		Freelancer affected by review.
+* date							Date			date of the review, default now.
+*
+* _id (ObjectID) will be added automatically by mongoose if not specified
+*/
+
+'use strict';
+const mongoose = require('mongoose');
+const ObjectID = mongoose.Schema.Types.ObjectId;
+require ('./Freelance');
+
+const Review = exports.Review = new mongoose.Schema({
+	//in future maybe add User and author is ObjectID ref User
+		author 	: { type: String, required: true },
+		title 	: { type: String },
+		text 	: { type: String },
```

Comment:
On GitHub indentation is sometimes weirded out, but this line and line 24 just afterwards are actually wrongly indented in the actual file.